shopfloor_delivery_shipment/services/delivery_shipment.py

```diff
@@ -663,7 +663,12 @@ def _find_move_lines_from_product(
     def _find_move_lines_not_loaded_from_shipment(self, shipment_advice):
         """Returns the move lines not loaded at all from the shipment advice."""
         domain = self._find_move_lines_domain(shipment_advice)
-        domain.append(("qty_done", "=", 0))
+        domain.extend(
+            [
+                ("qty_done", "=", 0),
+                ("picking_id.picking_type_id", "in", self.picking_types.ids),
```

Comment:
the picking type filter is already there in `_find_move_lines_domain`, n'est pas?

Reply:
tests?

Reply:
n'est-ce pas*

Reply:
then it fixes nothing, right?